resources/scripts/downloader.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
```

Comment:
issue(non-blocking): This is often considered a bad practice, see [this Stack Overflow post](https://stackoverflow.com/questions/19622198/what-does-set-e-mean-in-a-bash-script)